src/fuse_api.pxi

```diff
@@ -266,12 +266,15 @@ def main(workers=None):
     if *workers* is ``1``). These (and all worker threads) are guaranteed to
     have terminated when `main` returns.
 
-    While this function is running, special signal handlers will be installed
-    for the *SIGTERM*, *SIGINT* (Ctrl-C), *SIGHUP*, *SIGUSR1* and *SIGPIPE*
-    signals. *SIGPIPE* will be ignored, while the other three signals will cause
+    *handle_signals* specifies if special signal handlers should be installed
```

Comment:
That's very hard to read (runaway sentence). Can you just prefix the existing docstring with "Unless handle_signals is False, while this function is running [...]"?